packages/custom-storage/src/constants.ts

```diff
@@ -1,2 +1,3 @@
 export const DEFAULT_SERVER = 'https://{user}.carto.com/';
 export const DEFAULT_API_KEY = 'default_public';
+export const DEFAULT_MAX_RETRIES = 0;
```

Comment:
I have just removed this file in #57 (extracting the first 2 constants to the new core package). We can keep this file as it is, with just the new const or maybe extract it (to the CustomStorage.ts??)